lonestar/analytics/cpu/triangles/Triangles.cpp

```diff
@@ -403,12 +455,12 @@ void readGraph(Graph& graph) {
     std::string triangleFilename = inputFilename + ".triangles";
     std::ifstream triangleFile(triangleFilename.c_str());
     if (!triangleFile.good()) {
-      // triangles doesn't already exist, create it
+      // triangles doesn't already exist
       galois::StatTimer Trelabel("GraphRelabelTimer");
-      galois::gPrint("WARNING: Sorted graph does not exist; Relabelling and "
-                     "Creating a sorted graph\n");
+      galois::gPrint("WARNING: Sorted graph (.triangles) does not exist; Relabelling and "
```

Comment:
I'm in favor of dropping the option to load a sorted graph. It keeps this simple and consistent among benchmarks.

Reply:
No objections. @gurbinder533?

Reply:
I do have one comment though.

I mentioned this to you before Roshan, but we really should implement parallel input processing independent of however many threads the user decides to run the actual application for itself. I can see this being an issue if someone wants to check scaling and they're stuck with a single-threaded reordering scheme.

Maybe not for this upcoming release, but it should happen at some point.

Reply:
Good point. 

Not required for this release. But in the future, we can add a hidden option to specify number of threads for file loading.

Reply:
Dropped in local commit. If Gurbinder objects he may make his objections known later. :P